src/OrchardCore.Modules/OrchardCore.Search/Views/Search-Results.cshtml

```diff
@@ -1,13 +1,26 @@
+@using OrchardCore.ContentManagement
 @model SearchResultsViewModel
 @inject IContentItemDisplayManager DisplayManager
+@inject IContentManager ContentManager
 
 @if (Model.ContentItems != null && Model.ContentItems.Any())
 {
     <ul class="list-group">
         @foreach (var contentItem in Model.ContentItems)
         {
+            var metadata = await ContentManager.PopulateAspectAsync<ContentItemMetadata>(contentItem);
+
             <li class="list-group-item">
-                @await DisplayAsync(await DisplayManager.BuildDisplayAsync(contentItem, null, "Summary"))
+                @if (Model.Highlights is not null && Model.Highlights.TryGetValue(contentItem.ContentItemId, out var highlights))
```

Comment:
@sebastienros I am not sure how I feel about adding this condition in the view. The idea is to show the Hightlights if the request configured by the site admin returns highlights. Otherwise we display the contentItem in summary mode as we do today.
The use is able to override this view, but not sure how you feel about having it here by default.